tests/test_flow_studio_ui_ids.py

```
@@ -751,31 +751,45 @@ def test_run_detail_body_has_uiid(self):
 
     def test_run_detail_rerun_button_has_uiid(self):
         """Run detail modal re-run button should have data-uiid."""
-        html = get_flow_studio_html()
-        uiids = {uiid for uiid, _ in extract_uiids_from_html(html)}
+        # Read the TS file directly since modal content is generated dynamically
+        with open("swarm/tools/flow_studio_ui/src/run_detail_modal.ts", "r") as f:
```

Ruff (0.15.11)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)